extensions/resteasy-classic/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyBuiltinsProcessor.java

```diff
@@ -61,6 +61,7 @@ void setUpDenyAllJaxRs(CombinedIndexBuildItem index,
             List<String> resourceClasses = resteasyDeployment.getDeployment().getScannedResourceClasses();
             for (String className : resourceClasses) {
                 ClassInfo classInfo = index.getIndex().getClassByName(DotName.createSimple(className));
+                if(classInfo==null) throw new IllegalStateException("Unable to find class info for " + className);
```

Comment:
Can we also add instructions here on what to do when this happens?

Reply:
I don't know if know what that would be?

Reply:
Something like: `Make sure the class exists in the application or if it's in a separate JAR make sure it's indexed or contains a beans.xml`?

Reply:
ah yes, that would be an improvement.